packages/core/admin/admin/src/pages/ProfilePage.tsx

```diff
@@ -132,6 +132,11 @@ const ProfilePage = () => {
   >(
     async (body) => {
       const { confirmPassword: _confirmPassword, currentTheme, ...dataToSend } = body;
+
+      if (dataToSend.password === '') {
+        delete dataToSend.password;
+      }
```

Comment:
Can you explain why this is necessary as a code comment? Not knowing this is exactly how we got into this situation 😭